packages/cxl-lumo-styles/scss/typography.scss

```diff
@@ -12,7 +14,11 @@ html {
    *
    * @ see https://cdn.vaadin.com/vaadin-lumo-styles/1.5.0/demo/typography.html#font-size
    */
-  --cxl-lumo-font-size-hero: 4rem;
+  --cxl-lumo-font-size-hero: var(--lumo-font-size-xxxl);
+
+  @media #{mq.$medium} {
```

Comment:
Let's start at minimum surface, or `#{mq.$small}` and see how it works out. It should cover majority of phones (main problem targets).